content/en/synthetics/api_tests/_index.md

```diff
@@ -0,0 +1,177 @@
+| --------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `CONNRESET`       | The connection was abruptly closed by the remote server. Possible causes include the webserver encountering an error or crashing while responding, loss of connectivity of the webserver, etc. |
+| DNS             | DNS entry not found for the check URL. Possible causes include misconfigured check URL, wrong configuration of your DNS entries, etc.                                                          |
+| `INVALID_REQUEST` | The configuration of the check is invalid (e.g., typo in the URL).                                                                                                                             |
+| `SSL`             | The SSL connection couldn't be performed. [See the dedicated error page for more information][2].                                                                                     |
+| `TIMEOUT`         | The request couldn't be completed in a reasonable time.                                                                                                                                        |
+
+If a test fails, the uptime directly considers the endpoint as `down`. It is not re-tested until the next test run.
+
+### Notify your team
+
+A notification is sent according to the set of alerting conditions. To configure notifications:
+
+1. Select users and/or [services][3] to send the notifications to. Note that you can use the [`@-notification` feature][4] in the **message** field.
+2. Enter a **message** for the API test. This field allows standard [Markdown formatting][5]. Notification messages include the **message** defined in this section and information about which assertion failed and why.
+3. Click **Save** to save your API test.
+
+Notifications example:
+
+{{< img src="synthetics/api_tests/notifications-example.png" alt="API Test Notifications" responsive="true" style="width:80%;" >}}
+
+## Network timings
+
+The Synthetics details page displays the following network timings:
+
+| Timing                    | Description                                                                                           |
+|---------------------------|-------------------------------------------------------------------------------------------------------|
+| `DNS`                       | Time spent resolving the DNS name of the last request.                                                |
+| `Connect`                   | Time spent establishing a connection to the server.                                                   |
+| `SSL`                       | Time spent for the TLS handshake. If the last request is not over HTTPS, this metric does not appear. |
```

Comment:
missing the link to the ssl errors.

Reply:
the link is above in the Test Failure section :)